app/views/todos/index.html.erb

```diff
@@ -15,12 +15,35 @@
         <td><%= i+1 %></td>
         <td><%= todo.description %></td>
         <td class='days-left'><%= todo.days_left.ceil %></td>
-        <td><%= link_to "Delete", todo, method: :delete, class: 'btn btn-success' %></td>
+        <td><%= link_to "Delete", "javascript:void(0);", class: 'btn btn-success', data: { :"todo-id" => todo.id, :"delete-todo-button" => true } %></td>
       </tr>
     <% end %>
   </table>
-  <%= link_to "Add todo", new_todo_path, class: 'btn btn-success' %>
 <% else %>
-  <h3>No todos!</h3>
-  <%= link_to "Add todo", new_todo_path, class: 'btn btn-success' %>
+  <div id='no-todos'>
+    <h3>No todos yet.</h3>
+    Enter a description of your task in the box below.
+  </div>
+  <table class="table table-striped" id="todos">
+    <thead>
+      <h4>
+      <tr>
+        <td>#</td>
+        <td>Description</td>
+        <td>Days Left</td>
+        <td></td>
+      </tr>
+    </h4>
+    </thead>
+  </table>
+<% end %>
+<%= form_for @todos.new, :html => {:class => "form-horizontal center"} do |form| %>
```

Comment:
Passing `form_for` a symbol is fine when we're not working with a persisted instance (as is the case when starting a new Todo) - Rails has enough information from the symbol name to figure out what model is needed to back this form (e.g. `:todo` -> `class Todo`).

The API docs go into a bit more detail on using symbols and using model instances: http://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-form_for